code/modules/paperwork/paperplane.dm

```diff
@@ -87,9 +87,8 @@
 	add_fingerprint(user)
 
 
-/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0) //prevent the paper plane from spinning
-	if(!..())
-		return
+/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=FALSE, diagonals_first = FALSE)
+	. = ..(target, range, speed, thrower, spin=FALSE, diagonals_first)
```

Comment:
. = ..(target, range, speed, thrower, FALSE, diagonals_first)

you can't specify a named arg before a positional one (i.e after one argument of the form x=y, all future arguments must also be x=y

the other option is

    . = .. (target, range, speed, thrower, spin=FALSE, diagonals_first=diagonals_first)

but this is more confusing